resources/sql_scripts/CVManager_CreateTables.sql

```diff
@@ -324,6 +324,21 @@ SELECT ro.rsu_id, org.name
 FROM public.rsu_organization AS ro
 JOIN public.organizations AS org ON ro.organization_id = org.organization_id;
 
+-- Create iss keys table (id, iss_key, creation_date, expiration_date)

```

Comment:
Is this comment still relevant?

Reply:
Yes, we create the sequence for the primary key and then the table itself below.